syntax = "proto3";

package grpc_example;

service ExampleService {
    rpc SendMessage(RequestMessage) returns (ResponseMessage) {}
}

message RequestMessage {
    string message = 1;
}

message ResponseMessage {
    string reply = 1;
}
